doc/example/example.js

```diff
@@ -1,14 +1,15 @@
 /* global $, JitsiMeetJS */
+room_name = 'massmind';
```

Comment:
```suggestion
const room_name = 'massmind';
```
But I don't think that it will improve the readability that much if the related code is that far apart.